lib/brains/tests/test_management.py

```diff
@@ -0,0 +1,48 @@
+from django.core.management.base import CommandError
+
+from nose.tools import eq_, ok_, raises
+
+from lib.brains.management.commands import braintree_config as config
+from lib.brains.tests.base import BraintreeTest
+
+
+class TestManagement(BraintreeTest):
+
+    @raises(CommandError)
+    def test_bad(self):
+        config.import_file('foo.bar')
+
+    def test_good(self):
+        ok_(config.import_file('solitude.settings.products'))
+
+    def test_created(self):
+        seller = config.get_or_create_seller('uuid:fxa')
+        same_seller = config.get_or_create_seller('uuid:fxa')
+        eq_(seller, same_seller)
+
+    def test_created_product(self):
+        seller = config.get_or_create_seller('uuid:fxa')
+        seller_product = config.get_or_create_seller_product(
+            external_id='some:product:uuid:fxa',
+            public_id='some:product',
+            seller=seller)
+        eq_(seller_product, seller.product.get())
```

Comment:
what is this testing? Can the function name be more specific? I'm not sure what this assertion is proving.